### DISCUSSION TITLE
Building a Community of Contributors

### DISCUSSION THREAD

**Discussion by awaiken** [cardano-foundation/state-of-the-developer-ecosystem#10] [General feedback]
I understand the desire to gather all the technical information the survey is structured to do. Still, I think there is an element of non-technical questions that will help build longer-term engagement and more satisfied developers.  Some sample questions could be; what would make you feel more valued as a contributor to Cardano, how can Cardano or our ecosystem provide better support to you, why are you contributing to Cardano vs. other chains, how can we support your career aspirations, if you're building a dapp, how can we support your economic success (beyond grants), what are your long-term goals as a developer in the blockchain space? I've found it quite helpful getting beyond the purely technical to understand how to bring value to the developer and build consistent engagement.

**Comment by KtorZ** [comment]
I love the idea. I'll add this to the list of potential questions for 2025! Thanks.
